apps/tx-builder/package.json

```diff
@@ -4,12 +4,15 @@
   "private": true,
   "homepage": "/tx-builder",
   "dependencies": {
-    "@gnosis.pm/safe-deployments": "^1.11.0",
-    "@gnosis.pm/safe-react-components": "^1.1.4",
+    "@gnosis.pm/safe-apps-provider": "^0.11.3",
+    "@gnosis.pm/safe-apps-sdk": "^7.5.0",
```

Comment:
This is already updated in the global repo package.json. Does it need to be added here?

Reply:
It depends on how we want to approach the SDK <> App relation, if we use a global dependency then any time we update the SDK globally, we would also have to update the apps. But, on the other hand, if it has a local dependency that doesn't match the global version number, it will install the required version specially for the app.

Reply:
Hmm it looks like all the apps use the global version, will switch to the global here as well